src/main/kotlin/Archives/Archive.kt

```diff
@@ -0,0 +1,8 @@
+package Archives
+
+import Notes.Notes
+
+class Archive(
```

Comment:
🍏 Возможно, ты оценишь https://kotlinlang.org/docs/data-classes.html
```suggestion
data class Archive(
```